src/main/java/edu/ucsb/cs156/organic/services/SystemInfoServiceImpl.java

```diff
@@ -23,16 +28,30 @@ public class SystemInfoServiceImpl extends SystemInfoService {
   private boolean showSwaggerUILink;
 
   @Value("${app.sourceRepo}")
-  private String sourceRepo = "https://github.com/ucsb-cs156/proj-organic";
+  private String sourceRepo = "https://github.com/ucsb-cs156-s24/proj-organic-s24-5pm-4";
```

Comment:
I do need you to change this back.   Ultimately, the code that runs the app is proj-organic.

In terms of figuring out why localhost is different, consider this line:

https://github.com/ucsb-cs156-s24/proj-organic-s24-5pm-4/blob/44a929d1384a0cc88591765d5ae7e340817bb517/src/main/resources/application-development.properties#L13

I'm wondering if the fact that that is there in application-development.properties but not in application-production.properties is part of why it's not working on localhost?

Maybe experiment with that?